index.html

```diff
@@ -2,8 +2,8 @@
     <body>
         <p>Pay close attention!</p>
 
-        <p><a href="./badcode.html" target="_blank">This link was submitted by a community member</a></p>
+        <p><a href="./badcode.html" target="_blank" rel="noopener noreferrer">This link was submitted by a community member</a></p>
 
-        <button onclick="window.open('./badcode.html')">This link was submitted by a community member</button>
+        <button onclick="window.open('./badcode.html', 'next', 'noopener,noreferrer')">This link was submitted by a community member</button>
```

Comment:
- For javascript `window.open` function, add the values `noopener,noreferrer` in the `windowFeatures` parameter of the `window.open` function.